tests/unit/test_search.py

```diff
@@ -948,3 +961,51 @@ def test_no_matches(query_handler):
     # Invalid variation
     response = query_handler.search(variation='v600e')
     assert_no_match(response)
+
+
+def test_civic_id_search(query_handler, civic_vid33, civic_gid19,
+                         civic_tid146, civic_did8, eid2997_document):
+    """Test search on civic node id"""
+    res = query_handler.search_by_id(node_id='civic:vid33')
+    res = res['variation_descriptors']
+    assertions(civic_vid33, res)
+
+    res = query_handler.search_by_id(node_id='civic:gid19')
+    res = res['gene_descriptors']
+    assertions(civic_gid19, res)
+
+    res = query_handler.search_by_id(node_id='civic_tid146')
+    res = res['therapy_descriptors']
+    assertions(civic_tid146, res)
+
+    res = query_handler.search_by_id(node_id='civic_did8')
+    res = res['disease_descriptors']
+    assertions(civic_did8, res)
+
+    res = query_handler.search_by_id(node_id='civic_did8')
+    res = res['documents']
+    assertions(eid2997_document, res)
+
+
+def test_moa_id_search(query_handler, moa_vid69, moa_abl1, moa_imatinib,
+                       moa_chronic_myelogenous_leukemia, moa_aid69_document):
+    """Test search on moa node id"""
+    res = query_handler.search_by_id(node_id='moa:vid69')
+    res = res['variation_descriptors']
+    assertions(moa_vid69, res)
+
+    res = query_handler.search_by_id(node_id='moa.normalize.gene:ABL1')
+    res = res['gene_descriptors']
+    assertions(moa_abl1, res)
+
+    res = query_handler.search_by_id(node_id='moa.normalize.therapy:Imatinib')
+    res = res['therapy_descriptors']
+    assertions(moa_imatinib, res)
+
+    res = query_handler.search_by_id(node_id='moa.normalize.disease:Chronic%20Myelogenous%20Leukemia')  # noqa: E501
```

Comment:
Looks like this ID changed to using `CML`? I would double check why this is and make the appropriate change. It's causing tests to fail on my end.

Reply:
The ID is `Chronic%20Myelogenous%20Leukemia` cuz `Chronic Myelogenous Leukemia` has the highest match score from disease normalizer

Reply:
Hmmm.. I loaded my DB off this branch yesterday, but not sure if you made changes after. I'm loading again to see if it changes back to `Chronic%20Myelogenous%20Leukemia` instead of `CML`.